uni-5/03_V12_ALPHA_UPGRADE.md

```diff
@@ -0,0 +1,103 @@
+# Uni v12 Alpha Upgrade
+
+It's time to test Juno v12. Note that this upgrade introduces an oracle and price feeder, and these must now be run.
+
+To install:
+
+```sh
+cd ~
+git clone https://github.com/CosmosContracts/juno.git
+cd ./juno/price-feeder
+make install
+
+# Copy config to your location for future use & editing
+cp config.example.toml $HOME/.juno/oracle-config.toml
+
+# If you get `cp: cannot create regular file '$HOME/.juno/oracle-config.toml': No such file or directory`, 
+# run: `junod init [moniker]`. Then run the cp command again
+
+price-feeder version # sdk: v0.45.11 and go1.19.*
+# If it is not found, your go path is not set
+```
+
```

Comment:
```suggestion

Edit your juno home (~/.juno/config/app.toml) app.toml to be 0 fees for any denoms set which will take effect next restart.
minimum-gas-prices = "0ujunox"

Next, create a new price feeder account in the test keyring. This account will need to be sent 1 JUNOX _(1000000ujunox)_. This can be done before or after the upgrade takes place

junod keys add feeder --keyring-backend test
junod tx bank send <validator> <feeder0address> 1000000ujunox --chain-id uni-5 --fees 500ujunox

```